packages/postcss-reduce-initial/data/toInitial.json

```diff
@@ -17,17 +17,19 @@
   "box-sizing": "content-box",
   "column-rule-color": "currentcolor",
   "font-synthesis": "weight style",
+  "image-orientation": "from-image",
   "mask-clip": "border-box",
   "mask-mode": "match-source",
   "mask-origin": "border-box",
-  "mask-repeat": "repeat",
+  "mask-repeat": "no-repeat",
   "mask-type": "luminance",
   "ruby-align": "space-around",
   "ruby-merge": "separate",
+  "ruby-position": "alternate",
   "text-decoration-color": "currentcolor",
   "text-emphasis-color": "currentcolor",
   "text-emphasis-position": "over right",
-  "transform-box": "border-box",
+  "transform-box": "view-box",
   "transform-origin": "50% 50% 0",
   "vertical-align": "baseline",
   "writing-mode": "horizontal-tb"
```

Comment:
Will be great to look at git history of `toInitial.json` and `fromInitial.json`, some properties were removed due incompatibility between browsers

Reply:
Ah, so the files were edited by hand? Good to know!

Reply:
It seems that we did this once, so it's better to check, maybe it's worth leaving a comment about this so as not to forget next time

Reply:
Oh... yeah, good to know. But is that desirable to do anyway? 

If such manual overrides are necessary, would it perhaps be more appropriate to make changes to the script rather than the files themselves? So that one can update the content in a repeatable/predictable manner.

One solution could be to have an "override list" in a separate file that replaces the initial value from the MDN data before it's reduced and processed further. Then no extra logic beyond that would be necessary.

Having to examine the git history and try to determine what needs to be kept or not seems asking for trouble

Reply:
Something like...

```js
// lib/overrideInitial.mjs
const overrides = {
  'property-name-here': {
    comment: 'comment about why this propertys initial value is overriden',
    newInitial: 'foo',
  },
};

export default overrides;
```

Then add `overrideInitial` to the promise chain:

```js
return fetch(jsonURL)
    .then((result) => result.json())
    .then(overrideInitial)
    .then(reduceInitial)
    .then(validate)
```

And the `overrideInitial` function simply checks against the object and replaces any values that matches.

Then `overrideInitial.mjs` would also help document the overrides for the future

Reply:
I am starting to believe the changes to the JSON files should go in a separate pull request, as checking which replacements should be allowed is going to be quite time-intensive. We need to check that most browsers out there actually adhere to the spec, not just what's allowed in theory.

Reply:
Alright - sounds sensible!

But then I would propose including (as part of that new PR) an override mechanism along the lines of what I mention above - so that the manual changes can be tracked and documented in one place and always applied when the script is run to re-generate the files. Rather than manually applied each time.

To aid in house-keeping I could include the original value in the override file too, and print a warning to the console if it doesn't match up with the data fetched from MDN (i.e. might be stale). And do the same if the data from MDN suddenly matches the override, making it redundant.

Reply:
I guess splitting the two would make it clearer which browser compatibility workarounds the plugin uses.